api/src/functions/graphql.js

```diff
@@ -11,17 +11,10 @@ import { db } from 'src/lib/db'
 import { logger } from 'src/lib/logger'
 
 export const handler = createGraphQLHandler({
-  getCurrentUser,
```

Comment:
Hey @chrisvdm maybe this is the problem here—any reason why we're not passing `getCurrentUser` along?